src/transport/datamodel/utransport.rs

```diff
@@ -47,7 +47,7 @@ pub trait UTransport {
     /// On success, returns a tuple containing the received `UPayload` and `UAttributes` wrapped in `Ok(_)`.
     /// If receiving the message fails, it returns `Err(UStatus)`,
     /// where `UStatus` contains a `UCode` indicating the specific error or failure reason.
-    async fn receive(&self, topic: UUri) -> Result<(UPayload, UAttributes), UStatus>;
+    async fn receive(&self, topic: UUri) -> Result<UMessage, UStatus>;
```

Comment:
The doc comment still refers to the original types. Which also is a good example for why you should not mention the types of parameters in the doc comments at all (they are obvious from the generated documentation anyway) ;-)